.github/stale.yml

```diff
@@ -0,0 +1,7 @@
+# Issues or Pull Requests with these labels will never be considered stale
+exemptLabels:
+  "first-timers only"
```

Comment:
Also, should we include "good first patch"?  Or are you considering folding those into the "first-timers only" label?